lib/src/commands/create/create.dart

```diff
@@ -121,6 +121,12 @@ class CreateCommand extends Command<int> {
         'application-id',
         help: 'The bundle identifier on iOS or application id on Android. '
             '(defaults to <org-name>.<project-name>)',
+      )
+      ..addOption(
```

Comment:
Why is it an option and not a flag?

Reply:
Because I forgot that was a thing 😅

Changed to a flag!